pkgs/applications/window-managers/sway/lock-fprintd.nix

```
@@ -0,0 +1,80 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, scdoc
+, wayland-scanner
+, wayland
+, wayland-protocols
+, libxkbcommon
+, cairo
+, gdk-pixbuf
+, pam
+, glib
+, dbus
+, fprintd
+, unstableGitUpdater
+}:
+
+stdenv.mkDerivation rec {
+  pname = "swaylock-fprintd";
+  version = "unstable-20230130";
```

```suggestion
  version = "unstable-2023-01-30";
```

The format documented in the manual for unstable git versions is `unstable-YYYY-MM-DD` (this was my bad in the derivation I wrote...)